rat/__init__.py

```diff
@@ -101,11 +101,21 @@
 
 ## Likelihood
 
-Rat inherits from the regression languages. As such, the first line of a Rat
-program is the likelihood. This line is special because it can access named
-variables in the input dataframe.
+Every Rat program comes with a dataframe. The dataframe defines the data which
+the Rat model fits itself too.
```

Comment:
I really hate being a pedant but I think this should be 'to'.

Reply:
Nah nah, you're right. I think pedantic implies a bit over the top, this is just a good ol' fashion grammar fix.